kuma/javascript/src/document.jsx

```diff
@@ -100,13 +100,15 @@ function DocumentPage({ document }: DocumentProps) {
             <A11yNav />
             <Header document={document} />
             <main role="main">
-                <Titlebar title={document.title} document={document} />
-                <div className="full-width-row-container">
-                    <div className="max-content-width-container">
-                        <Breadcrumbs document={document} />
-                        <LanguageMenu document={document} />
+                <header>
+                    <Titlebar title={document.title} document={document} />
+                    <div className="full-width-row-container">
+                        <div className="max-content-width-container">
+                            <Breadcrumbs document={document} />
+                            <LanguageMenu document={document} />
+                        </div>
                     </div>
-                </div>
+                </header>
```

Comment:
I am torn about whether we should mark this up as a `header`. I kinda feel like in this case, the header is the section before this i.e. the section that contains the logo, navigation, authentication etc.

I am thinking we should instead mark this up with something like:

```
<div class="subheader">
...
</div>
```

What do you think?

Reply:
I hear you, but there can be multiple headers per page, e.g. article headers. I vote for leaving it as a header.